applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_dynamic_U_Pw_scheme.hpp

```diff
@@ -289,6 +289,30 @@ class NewmarkDynamicUPwScheme : public NewmarkQuasistaticUPwScheme<TSparseSpace,
         KRATOS_CATCH( "" )
     }
 
+    void CalculateRHSContribution(
+        Condition& rCurrentCondition,
+        LocalSystemVectorType& RHS_Contribution,
+        Element::EquationIdVectorType& EquationId,
```

Comment:
Also here I guess we should prefix the name with `r` to comply with the style guide. Also, I would use the plural form to indicate that there are multiple IDs.
```suggestion
        Element::EquationIdVectorType& rEquationIds,
```